server/models/notifications.js

```diff
@@ -0,0 +1,54 @@
+const getNotificationsModel = (sequelize, DataTypes) => {
+  const flags = {
+    freezeTableName: true
+  };
+
+  const NotificationShema = {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      allowNull: false,
+      defaultValue: DataTypes.UUIDV4,
+      unique: true
+    },
+    userId: {
+      type: DataTypes.UUID
+    },
+    message: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    isSeen: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
+      allowNull: false
+    },
+    sourceLink: {
+      type: DataTypes.UUID
+    }
```

Comment:
It has been changed to source

Reply:
If source refers to an `id`, why not indicate it in the name? It makes the code self-explanatory. You could name it `sourceId` or simply `articleId` since it refers to the article the notification is for. Please rename.